src/lib/hooks/useFocus/index.test.ts

```diff
@@ -0,0 +1,102 @@
+import { describe, expect, it } from 'vitest';
+import { render, screen, fireEvent, waitFor } from '@testing-library/svelte/svelte5';
+import { userEvent } from '@testing-library/user-event';
```

Comment:
Seeing the following error in my local environment. I confirmed everything is installed via `pnpm i`.

![Screenshot 2024-05-16 at 4 24 43 PM](https://github.com/skeletonlabs/floating-ui-svelte/assets/1509726/a7b5e02c-5542-4597-b023-714700c9a4bb)

Reply:
It's definitely there:
![image](https://github.com/skeletonlabs/floating-ui-svelte/assets/63101006/fc85223a-4da9-4dd0-8620-f45be79f3c76)
Try restarting your LSP